src/applications/apps.js

```diff
@@ -77,24 +77,24 @@ export function checkActivityFunctions(location) {
 
 export function getAppsToLoad() {
   return apps
-    .filter(shouldBeActive)
     .filter(notSkipped)
     .filter(isntLoaded)
+    .filter(shouldBeActive)
 }
 
 export function getAppsToUnmount() {
   return apps
-    .filter(shouldntBeActive)
     .filter(notSkipped)
     .filter(isActive)
+    .filter(shouldntBeActive)
 }
 
 export function getAppsToMount() {
   return apps
-    .filter(shouldBeActive)
     .filter(notSkipped)
     .filter(isntActive)
     .filter(isLoaded)
+    .filter(shouldBeActive)
```

Comment:
Same question here. What does moving it down give us?

Reply:
It saves us unnecessary work -- if we know that an application isn't active because it is skipped or it's in the wrong status, there's no need to call its activity function.

Reply:
👍